src/lib/core/components/visualizations/implementations/MapVisualization.tsx

```diff
@@ -417,14 +417,12 @@ function MapViz(props: VisualizationProps) {
           ) {
             const geoAggKey = facetVariableDetails[0].value;
             map[geoAggKey] = {
-              label: String(
-                sum(
-                  response.sampleSizeTable.find(
-                    (item) =>
-                      item.facetVariableDetails != null &&
-                      item.facetVariableDetails[0].value === geoAggKey
-                  )?.size
-                )
+              entityCount: sum(
+                response.sampleSizeTable.find(
+                  (item) =>
+                    item.facetVariableDetails != null &&
+                    item.facetVariableDetails[0].value === geoAggKey
+                )?.size
               ),
```

Comment:
Do you really need to use `sum` here? Maybe my eyes are deceiving me, but it looks like you are summing a single value.

Reply:
However, it's correct.  the `size` property is an array.  IIRC correctly if the facet variable was Sex, then the values in `size` would be for male and female.

Reply:
Ah, makes sense!